hotel-search-api/src/test/java/com/hotel/api/search/service/HotelRoomAvailabilityServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package com.hotel.api.search.service;
+
+import com.hotel.api.search.mappers.roomAvailability.RoomAvailabilityRequestBuilder;
+import com.hotel.api.search.model.HotelRoomAvailabilityRequest;
+import com.hotel.service.roomavailability.HotelRoomAvailabilityServiceGrpc;
+import com.hotel.service.roomavailability.RoomAvailabilityRequest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static com.hotel.service.util.ProtoBufUtil.safeSetProtoField;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class HotelRoomAvailabilityServiceTest {
+
+    @Mock
+    HotelRoomAvailabilityServiceGrpc.HotelRoomAvailabilityServiceBlockingStub hotelRoomAvailabilityServiceBlockingStub;
+
+    @Mock
+    private RoomAvailabilityRequestBuilder roomAvailabilityRequestBuilder;
+
+    @InjectMocks
+    private HotelRoomAvailabilityService hotelRoomAvailabilityService;
+
+    @Test
+    void getRoomAvailabilityServiceTest() {
+        HotelRoomAvailabilityRequest hotelRoomAvailabilityRequest = new HotelRoomAvailabilityRequest();
+        hotelRoomAvailabilityRequest.setHotelCode("354422");
+        hotelRoomAvailabilityRequest.setRoomCount(2);
+        hotelRoomAvailabilityRequest.setCheckInDate("2022-10-18");
+        hotelRoomAvailabilityRequest.setCheckOutDate("2022-10-25");
+        hotelRoomAvailabilityRequest.setCurrencyCode("EUR");
+        hotelRoomAvailabilityRequest.setLanguageCode("FRE");
+        hotelRoomAvailabilityRequest.setOccupancy(1);
+        hotelRoomAvailabilityRequest.setCountryCode("FR");
+        hotelRoomAvailabilityRequest.setNoOfNights(2);
```

Comment:
Can we move this to another class or public method as hotelRoomAvailabilityRequest is used all the above 3 test classes